testing/tests/loops.rs

```diff
@@ -214,3 +217,46 @@ fn test_for_vec_attr_range() {
     };
     assert_eq!(t.render().unwrap(), "1 2 3 4 5 6 ");
 }
+
+struct NotClonable<T: fmt::Display>(T);
+
+impl<T: fmt::Display> fmt::Display for NotClonable<T> {
+    fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
```suggestion
    fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
```

I know every other `impl fmt::Display` also have `<'_>`, but it's not needed :)

Reply:
Implicit elided lifetimes are going away, so I think we should move in the other direction.

Reply:
Wait, really? That's going to introduce a lot of `<'_>` in general then. Do you have a reference? :)

Reply:
See some discussion here: https://github.com/rust-lang/rust/issues/80165. I guess there isn't really a concrete timeline, but I am personally convinced that the eliding the lifetimes does more harm (in terms of being able to understand code) than good.

Reply:
Huh, interesting. I always removed `<'_>`, as I was expecting `clippy::needless-lifetimes` around the corner to lint that in a new version. Even Rust's own `impl fmt::Display` examples use just `fmt::Formatter`.

Though I do agree, that eliding lifetimes in paths do more harm. Well, guess I have a lot of refactoring to do, in a lot of projects :)

Reply:
I think `cargo fix --edition-idioms` should be able to fix most of it, but it didn't seem to work all that well in this case (maybe `cargo fix` works in general after denying the lint?).